site/gatsby-site/src/components/incidents/IncidentForm.js

```diff
@@ -16,6 +32,40 @@ export const schema = Yup.object().shape({
 function IncidentForm() {
   const { values, errors, handleChange, handleSubmit } = useFormikContext();
 
+  const [editorSimilarIncidentReports, setEditorSimilarIncidentReports] = useState([]);
+
+  const [similarReportsById, setSimilarReportsById] = useState([]);
+
+  const client = useApolloClient();
```

Comment:
Why use `client` directly instead of `useQuery`? (I think using the client directly bypasses the Apollo cache)

Reply:
Oh, I didn't know that. Thanks. Changed.